lib/ar2dto/converter.rb

```diff
@@ -2,6 +2,8 @@
 
 module AR2DTO
   class Converter
+    ALLOWED_TYPES = [Symbol, BigDecimal, Regexp, IO, Range, Time, Date, DateTime,
```

Comment:
Just curious, where did you take this list from?

How likely it is to change at some point?

Reply:
I took it from here https://github.com/rails/rails/blob/main/activesupport/lib/active_support/core_ext/object/json.rb

the classes that implement as_json and are not "complex" objects like arrays, hashes, objects, etc